lib/galaxy/tools/relabel_from_file.xml

```diff
@@ -137,23 +137,75 @@
     </tests>
     <help><![CDATA[
 
+========
+Synopsis
+========
+
+This tool changes names of datasets within a collection using identifiers from a supplied file. 
+
 .. class:: infomark
 
-        This tool will take an input list and a text file with new identifiers
-        and build a new list with the same datasets but these new identifiers.
-        The order and number of entries in the text file must match the order
-        of the items you want to rename in your dataset collection.
+This document uses terms *name*, *label*, and *identifier* interchangeably: they all mean the same thing - name of individual datasets inside a collection. 
```

Comment:
Name can be the outer collection's name as you see it in the history, or the dataset name, which is not visible in a collection. The only correct thing here is identifier, and the verb is `to label`.

Reply:
done